app/api/routes.py

```diff
@@ -171,9 +169,9 @@ def get_resources():
         ]
     except Exception as e:
         logger.error(e)
-        return standardize_response(None, [{"code": "bad-request"}], "bad request", 400)
+        return standardize_response(payload=None, status_code=400)
```

Comment:
I'm on the fence about whether this should actually be a `400` or `500` status code... If an exception occurs here, that's unexpected, right? Which means that it's an unexpected server error, which would be a 500 response....

And this is in `get_resources` right? So if they get this far, then it's not a 404 by any means... It should probably be a `500`. Open to discussion on this point